src/components/List/Item.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+
+import { Text } from "../Text";
+import { HomeIcon } from "../Icons";
+import { List } from ".";
+
+export default {
+  component: List.Item,
+} as ComponentMeta<typeof List.Item>;
+
+const Template: ComponentStory<typeof List.Item> = (args) => (
+  <List>
+    <List.Item {...args}>
+      <HomeIcon size="large" />
+      <Text>List component</Text>
+    </List.Item>
+  </List>
```

Comment:
question: Is it sensible to create a story for a single item so you must wrap it in List? Maybe one List story will be enough?